<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	padding: 0px;
	margin: 0px;
	border: 0px;
}
QPushButton {
	font: 900 16pt &quot;Satoshi Black&quot;;
	background-color: transparent;
 	border-radius: 10px;
	font-weight: bold;	
	color: #F4F7ED;
	text-align: left;
	padding: 5px 5px;
}
#NavBar {
    background-color: #2E6E65; 
}
QLabel {
	font: 900 10pt &quot;Satoshi Black&quot;;
	font-size: 20px;
	font-weight: bold;
	color: #2E6E65;
	qproperty-alignment: 'AlignLeft';
    qproperty-wordWrap: true;
	background: transparent;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="NavBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="DashBoardButton_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Images/resources.qrc">
          <normaloff>:/icons/icons/align-justify.svg</normaloff>:/icons/icons/align-justify.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="MainButtons_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="DashboardButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 900 20pt &quot;Satoshi Black&quot;;	
}
</string>
          </property>
          <property name="text">
           <string>Dashboard</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/layout-dashboard.svg</normaloff>:/icons/icons/layout-dashboard.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="TransactionsButton">
          <property name="text">
           <string>Transactions</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/tickets.svg</normaloff>:/icons/icons/tickets.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LabButton">
          <property name="text">
           <string>Lab Request</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/test-tubes.svg</normaloff>:/icons/icons/test-tubes.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Settings</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/user-round-cog.svg</normaloff>:/icons/icons/user-round-cog.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LogoutButton">
          <property name="text">
           <string>Logout</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/log-out.svg</normaloff>:/icons/icons/log-out.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="MainBodyContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#MainBodyContainer {
	background-color: #f0f0f0;
}

#Header {
	padding: 20px 20px;
}
QLabel {
    padding: 0px; 
}
#UserType {
	font: 300 10pt &quot;Lexend Light&quot;;
	font-size: 15px;
	color: #2B3752;
}
#UserName {
	font: 600 10pt &quot;Lexend SemiBold&quot;;
	font-size: 25px;
	color: #2B3752;
}
#InterfaceTitle{
	font-size: 40px;
}
QGraphicsView {
    border: 2px solid #ccc; 
    border-radius: 25px;
    background: white; 
}

QComboBox {
    background-color: #2E6E65; 
    font-size: 15px;
    color: #1b3b3a; 
    border-radius: 10px;
    padding: 5px 10px;
    border: 1px solid #2E6E65; 
}

QComboBox::drop-down {
    background: transparent;
    border: none;
    width: 20px;
}

QPushButton {
	background-color: #2e6e65;
	font-size: 20px;
}



</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="Header">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="InterfaceTitle">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="UserPanel">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="UserInfo">
             <property name="spacing">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="UserName">
               <property name="text">
                <string>Roy Adrian Rondina</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="UserType">
               <property name="text">
                <string>Nurse</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="Profile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="ProfilePic">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="TopOverview" native="true">
           <property name="styleSheet">
            <string notr="true">#AddCheckUpButton, #AddTransactionButton {
	text-align: center;
}

#TotalPatient, #DayFrame {
	border: 2px solid  #2e6e65;
    border-radius: 15px; 
}

#Review {
	color: #2b3752;
	font-weight: 300;
	font-size: 12px;
}

#Total, #Time {
	color: #2b3752;
	font-size: 33px;
}

#Day {
	color: #2e6e65;
	font-weight: 300;
	font-size: 12px
}

</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignBottom">
             <widget class="QPushButton" name="AddCheckUpButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string>Add Check Up</string>
              </property>
              <property name="icon">
               <iconset resource="../Images/resources.qrc">
                <normaloff>:/icons/icons/file-plus-2.svg</normaloff>:/icons/icons/file-plus-2.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QPushButton" name="AddTransactionButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string>Add Transaction</string>
              </property>
              <property name="icon">
               <iconset resource="../Images/resources.qrc">
                <normaloff>:/icons/icons/ticket-plus.svg</normaloff>:/icons/icons/ticket-plus.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="TotalPatient">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Total Patient</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Total">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>0000</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignTop">
                <widget class="QLabel" name="Review">
                 <property name="text">
                  <string>20 more than yesterday</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="DayFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item alignment="Qt::AlignRight">
                <widget class="QLabel" name="Time">
                 <property name="text">
                  <string>00:00 AM</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLabel" name="Day">
                 <property name="text">
                  <string>Sunday</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLabel" name="Month">
                 <property name="text">
                  <string>Month 00, 2025</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="TableFrame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Pending Check Ups</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QScrollArea" name="scrollArea">
                 <property name="styleSheet">
                  <string notr="true">QScrollBar:vertical {
     background: transparent;
     width: 10px;
	border-radius: 5px;
}
QScrollBar::handle:vertical {
    	background: #C0C0C0;
    	border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    	background: #A0A0A0;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical{
    	background: none;
    	border: none;
}</string>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>826</width>
                    <height>522</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>11</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QTableWidget" name="PendingTable">
                     <property name="styleSheet">
                      <string notr="true">QTableWidget {
	background-color: #F4F7ED;
	gridline-color: transparent;
	border-radius: 10px;
	border: 2px solid #2E6E65;
}
QTableWidget::item {
	border: none;
	font: 16pt &quot;Lexend&quot;;
}
QTableWidget::item:selected {
	background-color: rgba(46, 110, 101, 0.3);
}
QTableWidget QHeaderView::section {
	background-color: #2E6E65;
	color: white;
	padding: 5px;
	font: 18px &quot;Lexend Medium&quot;;
	border: 2px solid #2E6E65;
}</string>
                     </property>
                     <attribute name="horizontalHeaderStretchLastSection">
                      <bool>true</bool>
                     </attribute>
                     <column>
                      <property name="text">
                       <string>Check up ID</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Patient Name</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Check Up Type</string>
                      </property>
                     </column>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="LowerOverview">
                 <property name="styleSheet">
                  <string notr="true">#DiagnosisOverview {
	border: 2px solid  #2e6e65;
    border-radius: 15px; 
}

#AgeStageOverview{
	border: 2px solid  #2e6e65;
    border-radius: 15px; 
}

#ByDiagnosisLabel, #ByAgeStageLabel {
	color: #2b3752;
	font-weight: 300;
	font-size: 12px;
}

#ChildLabel, #AdultLabel, #ElderlyLabel, #TotalLabel {
	color: #2b3752;
	font-weight: 300;
	font-size: 20px;
}

#ChildPercentage, #AdultPercentage, #ElderlyPercentage, #TotalPercentage {
	color: #2b3752;
}

#Diagnosis1, #Diagnosis2, #Diagnosis3, #Diagnosis4 {
	color: #2b3752;
	font-weight: 300;
	font-size: 20px;
}

#DiagnosisPercentage1, #DiagnosisPercentage2, #DiagnosisPercentage3, #DiagnosisPercentage4 {
	color: #2b3752
}

#frame, #frame_2, #frame_5, #frame_7, #frame_8, #frame_9, #frame_10 {
	background-color: #b9cec4;
	border-radius: 13px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
